proto/s2s.go

```diff
@@ -20,6 +20,13 @@ type Job struct {
 	Data      map[string]interface{} `json:"data"`      // job-specific data during Job.Run
 	Retry     uint                   `json:"retry"`     // retry N times if first run fails
 	RetryWait uint                   `json:"retryWait"` // wait time (milliseconds) between retries
+	Sequence  *Sequence              `json:"sequence"`
```

Comment:
`// set if job begins or ends a sequence, else nil`

Reply:
This is actually set for all jobs. I'll update comment to say that.